src/main/scala/beam/calibration/utils/DeleteSuggestion.scala

```diff
@@ -28,68 +27,55 @@ object DeleteSuggestion extends LazyLogging {
     }
   }
 
-  def deleteSuggestions(experimentId: String, suggestions: List[String]) = {
-
+  def deleteSuggestions(experimentId: String, suggestions: List[String]): Unit = {
     BeamSigoptTuner.fetchExperiment(experimentId) match {
-      case Some(_experiment) => {
+      case Some(_experiment) =>
         suggestions.foreach { suggestionId =>
-          {
-
-            _experiment.suggestions().delete(suggestionId).call()
-          }
+          _experiment.suggestions().delete(suggestionId).call()
         }
-      }
-      case None => {
+      case None =>
         logger.info(s"Experiment with id $experimentId not found")
-      }
     }
   }
 
-  def listSuggestions(experimentId: String) = {
+  def listSuggestions(experimentId: String): Unit = {
     BeamSigoptTuner.fetchExperiment(experimentId) match {
-      case Some(_experiment) => {
-
-        if (_experiment.suggestions().list().call().getData.size() > 0)
-          _experiment.suggestions().list().call().getData.forEach { d =>
-            println(d)
-          } else
-          logger.info(s"Experiement with id $experimentId has no suggestion")
-      }
-      case None => {
+      case Some(_experiment) =>
+        val data = _experiment.suggestions().list().call().getData
+        data.forEach(println)
+        if (data.isEmpty) {
```

Comment:
I see what you did here, but for semantics, it still seems that it should be `if is empty log else print each` I'm fine either way.